src/prognostic_equations/edmfx_entr_detr.jl

```diff
@@ -254,10 +254,16 @@ function detrainment(
     ᶜRH⁰::FT,
     ᶜbuoy⁰::FT,
     dt::FT,
-    ::ConstantCoefficientDetrainment,
+    ::BOverWDetrainment,
 ) where {FT}
     detr_coeff = CAP.detr_coeff(params)
-    detr = min(detr_coeff * abs(ᶜwʲ), 1 / dt)
+    detr = min(
+        abs(ᶜwʲ) * (
+            detr_coeff +
+            FT(0.12) * abs(min(ᶜbuoyʲ - ᶜbuoy⁰, 0)) / max(eps(FT), (ᶜwʲ - ᶜw⁰) * (ᶜwʲ - ᶜw⁰))
```

Comment:
I will use the parameter instead of the hard coded `0.12` once I update dependencies.